cmd/minishift/cmd/services/services.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright (C) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package services
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var ServicesCmd = &cobra.Command{
+	Use:   "services SUBCOMMAND [flags]",
+	Short: "Manages services used by Minishift.",
+	Long:  `Manages services used by Minishift.`,
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
You should help `cmd.Help()` in the Run. As this will enable users to see the usage after running `$ minishift services`

Reply:
these commands are NOT exposed to the user... they are internal commands that are invoked by forking the minishift process; like `sftpd`, `dns`, `proxy` and `systemtray`. they are not user facing

Reply:
```
Available Commands:
  addons      Manages Minishift add-ons.
  completion  Outputs minishift shell completion for the given shell (bash or zsh)
  config      Modifies Minishift configuration properties.
  console     Opens or displays the OpenShift Web Console URL.
  delete      Deletes the Minishift VM.
  docker-env  Sets Docker environment variables.
  help        Help about any command
  hostfolder  Manages host folders for the Minishift VM.
  image       Exports and imports container images.
  ip          Gets the IP address of the running cluster.
  logs        Gets the logs of the running OpenShift cluster.
  oc-env      Sets the path of the 'oc' binary.
  openshift   Interacts with your local OpenShift cluster.
  profile     Manages Minishift profiles.
  ssh         Log in to or run a command on a Minishift VM with SSH.
  start       Starts a local OpenShift cluster.
  status      Gets the status of the local OpenShift cluster.
  stop        Stops the running local OpenShift cluster.
  update      Updates Minishift to the latest version.
  version     Gets the version of Minishift.
```

`services` is not shown on purpose as they are used like deamon services.